packages/app/auth/AuthLoader.tsx

```diff
@@ -0,0 +1,19 @@
+import { useUserQuery } from './hooks';
+
+export const AuthLoader = ({
+  children,
+  loadingElement,
+  unauthorizedElement,
+}) => {
+  const { user, isLoading } = useUserQuery();
```

Comment:
i think the data coming back from here is what is causing the issue on native side. works fine on web but i'm having issues on ios